cogs/clash_river_war.py

```diff
@@ -139,6 +139,30 @@ def remaining_war_embed(all_clan_attacks):
     embed.add_field(name=columns, value=row_val, inline=False)
     return embed
 
+def players_war_attacks(all_current_war_players: PlayersRemainingWarAttacks):
+    embed = discord.Embed(
+        description=dedent('''
+            **Current River Race Players War Attacks **
+            **decks Used Today** - How many decks the player used today (maximum of 4)
+            **In Clan** - Checking whether is currently in the clan
+            **Player Name** - Accumulated Medals throughout the River Race
+            Note: This list filters all players who has done all their current war
+        '''.format()),
+        colour=discord.Colour.green()
+    )
+
+    columns = '**Decks Used Today**|**In Clan**|**Name**'
+    row_val = []
+    for current_war_player in all_current_war_players:
+        row_val.append(' `{:^5}` | `{:^1}` | **`{}`**'.format(
+            current_war_player.decks_used_today,
+            'T' if current_war_player.in_clan else 'F',

```

Comment:
Nicer to read with `X` for False and `/` for True? Or something even clearer is `Y` and `N`

Reply:
Done